virtualizarr/xarray.py

```diff
@@ -58,6 +59,8 @@ def open_virtual_dataset(
         Type of file to be opened. Used to determine which kerchunk file format backend to use.
         Can be one of {'netCDF3', 'netCDF4', 'HDF', 'TIFF', 'GRIB', 'FITS', 'zarr_v3'}.
         If not provided will attempt to automatically infer the correct filetype from header bytes.
+    group : str, default is None
+        Specific HDF Group in file to open (others ignored).
```

Comment:
Just making this a bit more similar to the description in the [docstring of `xr.open_dataset`](https://docs.xarray.dev/en/stable/generated/xarray.open_dataset.html)

```suggestion
        Path to the HDF5/netCDF4 group in the given file to open. Given as a str, supported by filetypes “netcdf4” and “hdf5”.
```